### PR TITLE
Localize content/en/docs/reference/glossary/disruption.md into Hindi

### LINKED ISSUE #37513: [hi] Localize /en/docs/reference/glossary/disruption.md
### Prerequisites:

- If this is your first contribution, visit [Contribute to K8s docs](https://kubernetes.io/docs/contribute/) for information on how to get started.
- Review a few https://github.com/kubernetes/website/issues/35600#issuecomment-1214215859 to see which 
  content you have to change and which you leave unchanged.

### Procedure:

- Make sure your base, target branch is `main` and create a working branch called `hi-disruption.md`.
- Create a file called content/hi/docs/reference/glossary/disruption.md.
- Copy the content of 
  https://github.com/kubernetes/website/blob/main/content/en/docs/reference/glossary/disruption.md
  to the file you created.
- Localize the content.
- Push your change and create a pull request.

Completing this feature issue fixes part of https://github.com/kubernetes/website/issues/35600

> **Note**:- Don't forget to join [`slack`](https://communityinviter.com/apps/kubernetes/community) `kubernetes-docs-hi` channel if facing any problem!
 

/language hi
/triage accepted

### PR DESCRIPTION
**Pull request by sandeepkanabar**
The PR localizes content/en/docs/reference/glossary/disruption.md into Hindi.

Fixes #37513

### CODE REVIEW

File: content/hi/docs/reference/glossary/disruption.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+short_description: >
+   ऐसी घटना जिसके कारण पॉड(पॉड्स) सेवा से बाहर हो जाते हैं।
+aka:
+tags:
+- fundamental
+---
+ विघटन वे घटनाएं हैं जिनसे एक या अधिक 
+{{< glossary_tooltip term_id="pod" text="पॉड्स" >}} सेवा से बाहर हो जाते हैं। 
+विघटन का असर उन संसाधनों पर भी पड़ता है, जैसे कि 
+{{< glossary_tooltip term_id="deployment" >}}, जो प्रभावित पॉड्स पर निर्भर करते हैं।
```
Comment (kundan2707) [inline content/hi/docs/reference/glossary/disruption.md:15]:
```suggestion
विघटन का असर उन संसाधनों पर भी पड़ता है जो प्रभावित पॉड्स पर निर्भर करते हैं, जैसे कि 
{{< glossary_tooltip term_id="deployment" >}}
```

Comment (sandeepkanabar) [reply content/hi/docs/reference/glossary/disruption.md]: Thanks. Accepted the suggestion.

File: content/hi/docs/reference/glossary/disruption.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+---
+ विघटन वे घटनाएं हैं जिनसे एक या अधिक 
+{{< glossary_tooltip term_id="pod" text="पॉड्स" >}} सेवा से बाहर हो जाते हैं। 
+विघटन का असर उन संसाधनों पर भी पड़ता है जो प्रभावित पॉड्स पर निर्भर करते हैं, जैसे कि 
+{{< glossary_tooltip term_id="deployment" >}}
+
+<!--more-->
+
+अगर आप क्लस्टर ऑपरेटर के रूप में, किसी एप्लिकेशन से संबंधित पॉड को नष्ट करते हैं, 
+तो कुबेरनेट इसे _स्वैच्छिक विघटन_ कहता है। अगर कोई पॉड नोड फेलियर या व्यापक क्षेत्र
```
Comment (divya-mohan0209) [inline content/hi/docs/reference/glossary/disruption.md:20]:
```suggestion
तो कुबेरनेट्स इसे _स्वैच्छिक विघटन_ कहता है। अगर कोई पॉड नोड फेलियर या व्यापक क्षेत्र 
```

File: content/hi/docs/reference/glossary/disruption.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+ विघटन वे घटनाएं हैं जिनसे एक या अधिक 
+{{< glossary_tooltip term_id="pod" text="पॉड्स" >}} सेवा से बाहर हो जाते हैं। 
+विघटन का असर उन संसाधनों पर भी पड़ता है जो प्रभावित पॉड्स पर निर्भर करते हैं, जैसे कि 
+{{< glossary_tooltip term_id="deployment" >}}
+
+<!--more-->
+
+अगर आप क्लस्टर ऑपरेटर के रूप में, किसी एप्लिकेशन से संबंधित पॉड को नष्ट करते हैं, 
+तो कुबेरनेट इसे _स्वैच्छिक विघटन_ कहता है। अगर कोई पॉड नोड फेलियर या व्यापक क्षेत्र 
+में आउटेज के कारण ऑफलाइन हो जाता है, तो कुबेरनेट इसे _अस्वैच्छिक विघटन_ कहता है।
```
Comment (divya-mohan0209) [inline content/hi/docs/reference/glossary/disruption.md:21]:
```suggestion
में आउटेज के कारण ऑफलाइन हो जाता है, तो कुबेरनेट्स इसे _अस्वैच्छिक विघटन_ कहता है।
```

**Review by divya-mohan0209 — CHANGES_REQUESTED** [review]
nits; rest lgtm!

File: content/hi/docs/reference/glossary/disruption.md (hunk 2 above)
Comment (sandeepkanabar) [reply content/hi/docs/reference/glossary/disruption.md]: Thanks @divya-mohan0209 for the suggestion to use `कुबेरनेट्स`. Accepted the changes.

**Review by divya-mohan0209 — COMMENTED** [review]
/lgtm

**Review by dipesh-rawat — APPROVED** [review]
Thanks!

/approve

**Pull request merged**